Lib/test/test__opcode.py

```diff
@@ -82,12 +82,16 @@ def test_specialization_stats(self):
             self.assertCountEqual(stats.keys(), specialized_opcodes)
             self.assertCountEqual(
                 stats['load_attr'].keys(),
-                stat_names + ['fails'])
+                stat_names + ['specialization_failure_kinds'])
             for sn in stat_names:
                 self.assertIsInstance(stats['load_attr'][sn], int)
-            self.assertIsInstance(stats['load_attr']['specialization_failure_kinds'], tuple)
+            self.assertIsInstance(
+                stats['load_attr']['specialization_failure_kinds'],
+                tuple)
             for v in stats['load_attr']['specialization_failure_kinds']:
                 self.assertIsInstance(v, int)
+        else:
+            print('No specialization stats to check')
```

Comment:
Personally, I don't like tests that print stuff when they pass.

Reply:
Yeah. I'll remove that.